docs/declarative-customization/view-formatting.md

```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
+      },
+      "children": [
+        {
+          "elmType": "img",
+          "attributes": {
+            "src": "=if(@group.fieldData == 'California', 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/Flag_of_California.svg/1920px-Flag_of_California.svg.png',
+                    if(@group.fieldData == 'Chicago', 'https://upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_Chicago%2C_Illinois.svg/1920px-Flag_of_Chicago%2C_Illinois.svg.png',
+                    if(@group.fieldData == 'New York', 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Flag_of_New_York_City.svg/2560px-Flag_of_New_York_City.svg.png',
+                    if(@group.fieldData == 'Seattle', 'https://upload.wikimedia.org/wikipedia/en/thumb/6/6d/Flag_of_Seattle.svg/1920px-Flag_of_Seattle.svg.png',
+                    if(@group.fieldData == 'Washington DC', 'https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Flag_of_the_District_of_Columbia.svg/2560px-Flag_of_the_District_of_Columbia.svg.png',
+                    ''))))"
+          },
+          "style": {
+            "max-width": "24px",
+            "max-height": "24px",
+            "margin-top": "2px",
+            "border-radius": "2px"
+          }
+        },
+        {
+          "elmType": "div",
+          "children": [
+            {
+              "elmType": "span",
+              "style": {
+                "padding": "5px 5px 5px 5px",
+                "font-weight": "500"
+              },
+              "txtContent": "@group.fieldData.displayValue"
+            }
+          ]
+        },
+        {
+          "elmType": "div",
+          "forEach": "aggregate in @aggregates",
+          "children": [
+            {
+              "elmType": "div",
+              "style": {
+                "display": "=if([$aggregate.columnDisplayName] == 'Approved' && Number([$aggregate.value]) < @group.count, 'flex', 'none')",
+                "flex-direction": "row",
+                "justify-content": "center"
+              },
+              "children": [
+                {
+                  "elmType": "div",
+                  "txtContent": "='has approval pending for ' + Number(@group.count - Number([$aggregate.value])) + if(@group.count - Number([$aggregate.value]) > 1 , ' employees', ' employee')",
+                  "style": {
+                    "padding": " ",
+                    "font-weight": "500"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+
+```
+
+#### Example: Custom group footer
+
+In the example below we have list with group footer formatted according to aggregate value.
+![Employee list grouped by City with formatted group footer](../images/group-footer-list-layout.png)
+
+In this example the `footerFormatter` for `groupProps` is set to format the group footer and the `@columnAggregate` is used to access column aggregate.

```

Comment:
In this example the `footerFormatter` for `groupProps` are used to format the group footer and the `@columnAggregate` is used to access column aggregate.